packages/@aws-cdk/aws-cloudtrail/package.json

```diff
@@ -88,8 +88,14 @@
     "@aws-cdk/aws-s3": "^0.35.0",
     "@aws-cdk/cdk": "^0.35.0"
   },
+  "awslint": {
+    "exclude": [
+      "duration-prop-type:@aws-cdk/aws-cloudtrail.TrailProps.cloudWatchLogsRetentionTimeDays",
```

Comment:
Should we just call this `cloudWatchLogsRetention`?

Reply:
I reckon we should!